packages/cms-lib/personalAccessKey.js

```diff
@@ -127,16 +139,11 @@ const personalAccessKeyPrompt = async () => {
   const { name } = await promptUser(PERSONAL_ACCESS_KEY_FLOW);
   const portalId = getPortalId(name);
   const env = getEnv(name);
+  const websiteOrigin = getHubSpotWebsiteOrigin(env);
```

Comment:
Instead of `getHubSpotWebsiteOrigin` could we come up with a better name? Maybe `getHubSpotAppDomain`?